herc/config.py

```diff
@@ -0,0 +1,53 @@
+import pyhocon
+import threading
+import traceback
+import sys
+import importlib
+
+_config = None
+_configlock = threading.Lock()
+
+_configlocs = [ '/etc/herc.conf', 'herc.conf']
+
+def load_config():
+    """Attempts to load Herc config file."""
+    global _config
+    excs = []
+    with _configlock:
+        if _config is None:
+            for conf in _configlocs:
+                try:
+                    _config = pyhocon.ConfigFactory.parse_file(conf)
+                    break
+                except BaseException as be:
+                    excs.append(be)
+
+        if _config is None:
+            #still? must have failed
+            print("Fatal error - failed to load Herc config file!")
+            print("Locations checked:", _configlocs)
+            for exc in excs:
+                print("Errors, in order:")
+                traceback.print_tb(exc)
+            sys.exit(1)
+
+def get(*args, **kwargs):
+    """Gets a value from the config file."""
+    load_config()
+    return _config.get(*args, **kwargs)
+
+def importclass(classpath):
```

Comment:
counter argument: YAGNI?

I mean, I'm not greatly opposed to it though if it's only called in once place I'd say put it in that place until it's called in more than 1 place